pydy/codegen/tests/test_code.py

```diff
@@ -21,7 +21,7 @@ def test_rhs_args(self):
         sys = n_link_pendulum_on_cart(3, True, True)
 
         constants = sys.constants_symbols
-        specifieds = sys.specifieds_symbols
+        specifieds = list(sys.specifieds_symbols)
```

Comment:
Why is this necessary? `sys.specifieds_symbols` is automatically generated and should be a list of SymPy functions (of time).